cpp/test/GetProcessIdsForPath_test.cc

```diff
@@ -0,0 +1,26 @@
+// Test harness for ProcessUtil::GetProcessIdsForPath.
+#include "FileUtil.h"
+#include "ProcessUtil.h"
+#include "util.h"
+
+
+int Main(int, char **) {
+    const std::string TEMPFILE_NAME("/tmp/GetProcessIdsForPath_test.temp");
```

Comment:
use FileUtil::AutoTempFile instead?

Reply:
I intentionally did not use it to make what's happening more explicit.